pkg/util/filehelper/file_test.go

```diff
@@ -25,7 +25,7 @@ import (
 
 func Test_non_existent_directory_returns_false(t *testing.T) {
 	path := filepath.Join("this", "path", "really", "should", "not", "exists", "unless", "you", "have", "a", "crazy", "setup")
-	if Exists(path) == true {
```

Comment:
👍

Reply:
although identical in use, in some cases this construct is actually easier to reason about